### PR TITLE
Implemented support for the fee-0.6 EPP extension

### PR DESCRIPTION
**Pull request by wbond**
This is used for `nic.co`.

### CODE REVIEW

File: check.go [hunk 1]
```diff
@@ -55,12 +56,20 @@ func encodeDomainCheck(buf *bytes.Buffer, domains []string, extFee bool) error {
...
 	buf.WriteString(`</check>`)
 
-	if extFee {
+	supportsFee05 := greeting.SupportsExtension(ExtFee05)
+	supportsFee06 := greeting.SupportsExtension(ExtFee06)
+
+	if supportsFee05 || supportsFee06 {
+		feeURN := ExtFee06
+		if supportsFee05 {
+			feeURN = ExtFee06
```
Comment (cee-dub) [inline check.go]: This looks like it should assign `feeURN = ExtFee05`

File: check.go [hunk 2]
```diff
@@ -161,8 +170,7 @@ func init() {
 		return nil
 	})
 
-	// Scan fee-0.5 extension into Charges
-	path = "epp > response > extension > " + ExtFee + " chkData"
+	path = "epp > response > extension > " + ExtFee05 + " chkData"
```
Comment (cee-dub) [inline check.go]: Can you comment to explain why this path is always `ExtFee05` regardless of the support determined above?

File: check.go [hunk 3]
```diff
@@ -190,6 +198,31 @@ func init() {
 		return nil
 	})
 
+	path = "epp > response > extension > " + ExtFee06 + " chkData"
```
Comment (cee-dub) [inline check.go]: Sorry, had I read further, it would have been more obvious that we scan both paths.

Comment (wbond) [inline check.go]: Yeah, the XML structure is just slightly different, so I set up the scanner to handle whichever is present

**Comment by case** [comment]
This ready to merge, @ydnar?

**Pull request merged**